doc/api.md

```diff
@@ -104,6 +104,7 @@ Unofficial Twitter command. It shows all direct answers (excluding the original
 * max_id: maximum id
 * getText: Defines the format of the status field. Can be "html" or "plain"
 * include_entities: "true" shows entities for pictures and links (Default: false)
+* friendica_verbose: "true" enables different error returns for Windows 10 app (default: "false")

```

Comment:
I'm in my nitpicker mode again :-)

The "verbose" setting isn't for the windows 10 app but for every app that wants to have additional return values ;-)